src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchControllerTest.kt

```diff
@@ -62,7 +62,7 @@ class OffenderSearchControllerTest {
 
   @Test
   fun offenderSearch() {
-    wireMock.stubFor(WireMock.get(WireMock.anyUrl()).willReturn(
+    wireMock.stubFor(WireMock.post(WireMock.anyUrl()).willReturn(
```

Comment:
interestingly this version must have switched to using POST rather than GET (which IMHO is correct, always off doing a GET with a BODY)